### PR TITLE
Adding Missing Number and Is Even or Odd Snippet

### PR DESCRIPTION
**Pull request by TangoBeee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Inspired by [30 seconds of code](https://github.com/Chalarangelo/30-seconds-of-c
 [![Check if all elements of array are equal](https://img.shields.io/badge/-Check%20if%20all%20elements%20of%20array%20are%20equal-e1b050)](#check-if-all-elements-of-array-are-equal) [![link](https://img.shields.io/badge/-Repository%20link-969c56?logo=github)](https://github.com/iluwatar/30-seconds-of-java/blob/master/src/main/java/array/AllEqualSnippet.java)  
 [![Find maximum integer from the array](https://img.shields.io/badge/-Find%20maximum%20integer%20from%20the%20array-e1b050?)](#find-maximum-integer-from-the-array) [![link](https://img.shields.io/badge/-Repository%20link-969c56?logo=github)](https://github.com/iluwatar/30-seconds-of-java/blob/master/src/main/java/array/FindMaxSnippet.java)
 
+### bitmanipulation
```
Comment (iluwatar) [inline README.md:22]:
```suggestion
### Bit Manipulation
```

**Review by iluwatar — CHANGES_REQUESTED** [review]
We also need to address the failing build and add tests.

File: README.md (hunk 1 above)
Comment (TangoBeee) [reply README.md:22]: ah...
sry but what do you mean by that?

Comment (iluwatar) [reply README.md:22]: Just proposing to change the header

Comment (TangoBeee) [reply README.md]: done
and really sorry for inconvenience

i am having exams thats why :/

**Review by iluwatar — CHANGES_REQUESTED** [review]
The build is failing. Please investigate.

**Pull request closed without merging**